pages/features.js

```diff
@@ -53,8 +53,8 @@ export default function FeaturesPage() {
             </Icon>
             <Name>Storage agnostic</Name>
             <Description>
-              Use S3, Azure, GCP, SSH, SFTP, Aliyun OSS rsync or any
-              network-attached storage to store data. The list of supported
+              Use S3, Azure, Google Drive, GCP, SSH, SFTP, Aliyun OSS rsync or
+              any network-attached storage to store data. The list of supported
```

Comment:
Is the order important? I see you place "Google Drive" later in the list (before OSS) in the installation docs: 
```
`[s3]`, `[ssh]`, `[gs]`, `[azure]`, `[gdrive]`, and `[oss]`
```
Which I'm not sure is the best order (should probably start with SSH?) but still, having a consistent ordering may be desirable?

Reply:
@jorgeorpinel I haven't found order consistency in different places as well and just selected `azure` as pretty popular/import storage type to support and just searched whole website for its mentions and decided to put `Google Drive` after it :)

Reply:
Ok, this doesn't matter that much. I'll review it later.

Reply:
Addressed! In 63a4ac9c